src/warnet/server.py

```diff
@@ -325,8 +326,22 @@ def scenarios_available(self) -> list[tuple]:
                 module_name = f"warnet.scenarios.{s.name}"
                 try:
                     m = importlib.import_module(module_name)
-                    if hasattr(m, "cli_help"):
-                        scenario_list.append((s.name, m.cli_help()))
+                    for _name, obj in m.__dict__.items():
+                        if (
+                            isinstance(obj, type)
+                            and issubclass(obj, WarnetTestFramework)
+                            and obj != WarnetTestFramework
+                        ):
+                            scenario = obj()
+                            help_info = scenario.cli_help()
+                            scenario_list.append(
+                                {
+                                    "name": s.name,
+                                    "help_text": help_info["help_text"],
+                                    "options": help_info["options"],
+                                }
+                            )
+                            break  # Assuming one scenario class per module
```

Comment:
is this something we can enforce?

Reply:
We can really only call a single TestFrameWork class per scenario, so I don't think it should be a big issue?